bitcoinology/what-is-lightning-network.md

```diff
@@ -0,0 +1,90 @@
+
+## The Block Size War
+
+ So as we reach the end of 2016, it turns out there's a bunch of people actually starting to develop this thing called the Lightning Network, at least as software. Nobody's using it yet because this SegWit thing hadn't yet activated and so it was kind of like not very realistic to actually build it. Plus, because It was complicated, as I just explained to you. It is definitely a complicated system. Actually building out this software, LND, see lightning with Blockstream, Async, built Eclair, I'll talk about these guys in a minute. Building that software took time. Even getting just up to a first basic version took some time. So you had these companies or projects spring up. There was Lightning Labs, started by Elizabeth Stark here, along with LouLou, and they built this LND software. And this software will be behind a lot of the wallets that we're using later today. Blockstream, who started in 2015 with a whole sort of sidechain business. Rusty Russell, who's originally a Linux developer, and Christian Decker, who did a PhD in Bitcoin, sort of famously the first Bitcoin PhD. They were two of the people, and there were some other people who worked on another version of Lightning called C Lightning, and as I said before, Asank worked on Eclair. So you had at least three, and arguably more, different versions of this Lightning software being developed in 2016. And it got to the point, so by the end of 2016, we had the whole block size war raging. And in the background, at the same time, we had these guys putting together a set of protocols and agreements, because they had to make all of this complicated software talk to each other, and it was not trivial, yeah? I remember this, I put here the scaling Bitcoin sign, because in Milan in 2016, there was one of a series of scaling Bitcoin conferences, and one of the things that happened there was that was the first meetup. All these guys met up in a room. I remember there was a whole set of like 13 or 14 people, developers, and they were there solidly for two days just hashing it out, and what did they come up with? They came up with what's called nowadays the bolts.
```

Comment:
```suggestion
 So as we reach the end of 2016, it turns out there's a bunch of people actually starting to develop this thing called the Lightning Network, at least as software. Nobody's using it yet because this SegWit thing hadn't yet activated and so it was kind of like not very realistic to actually build it. Plus, because It was complicated, as I just explained to you. It is definitely a complicated system. Actually building out this software, LND, see lightning with Blockstream, Acinq, built Eclair, I'll talk about these guys in a minute. Building that software took time. Even getting just up to a first basic version took some time. So you had these companies or projects spring up. There was Lightning Labs, started by Elizabeth Stark here, along with LouLou, and they built this LND software. And this software will be behind a lot of the wallets that we're using later today. Blockstream, who started in 2015 with a whole sort of sidechain business. Rusty Russell, who's originally a Linux developer, and Christian Decker, who did a PhD in Bitcoin, sort of famously the first Bitcoin PhD. They were two of the people, and there were some other people who worked on another version of Lightning called C Lightning, and as I said before, Asank worked on Eclair. So you had at least three, and arguably more, different versions of this Lightning software being developed in 2016. And it got to the point, so by the end of 2016, we had the whole block size war raging. And in the background, at the same time, we had these guys putting together a set of protocols and agreements, because they had to make all of this complicated software talk to each other, and it was not trivial, yeah? I remember this, I put here the scaling Bitcoin sign, because in Milan in 2016, there was one of a series of scaling Bitcoin conferences, and one of the things that happened there was that was the first meetup. All these guys met up in a room. I remember there was a whole set of like 13 or 14 people, developers, and they were there solidly for two days just hashing it out, and what did they come up with? They came up with what's called nowadays the bolts.
```